package.json

```diff
@@ -11,10 +11,11 @@
     "url": "https://github.com/FuelLabs/sway-vscode-plugin.git"
   },
   "scripts": {
-    "vscode:prepublish": "npm run compile",
-    "compile": "tsc -b",
-    "watch": "tsc -b -w",
-    "test": "sh ./scripts/e2e.sh",
+    "vscode:prepublish": "npm run esbuild-base -- --minify",
+    "esbuild-base": "esbuild ./client/src/main.ts --bundle --outfile=client/out/main.js --external:vscode --format=cjs --platform=node",
+    "esbuild": "npm run esbuild-base -- --sourcemap",
+    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
+    "pre:install-extension": "rm sway-vscode-plugin.vsix",
```

Comment:
What is this used for?

Reply:
It looks like, the packing process will hang if a `vsix` already exists in your file system, so remove it before running the command `install-extension`